parsers/cron_file_parser.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+"""Simple parsers for cron type files."""
+
+
+from grr.lib import parsers
+from grr.lib import rdfvalue
+from grr.lib import utils
+from crontab import CronTab
+
+
+class CronTabParser(parsers.FileParser):
+  """Parser for crontab files."""
+
+  output_types = ["CronTabFile"]
+  supported_artifacts = ["OSXCronTabs", "LinuxCronTabs"]
+
+  def Parse(self, stat, file_object, knowledge_base):
+    """Parse the crontab file."""
+    _ = knowledge_base
+    entries = []
+
+    crondata = file_object.read(100000)
+    jobs = CronTab(tab=crondata)
+
+    for job in jobs:
+      minute = utils.SmartStr(job.minute)
```

Comment:
Initially did that for clarity, but I don't think it adds much so I will remove.